crates/bevy_audio/src/audio_output.rs

```diff
@@ -74,8 +89,99 @@ where
     let world = world.cell();
     let audio_output = world.get_non_send::<AudioOutput<Source>>().unwrap();
     let mut audio = world.get_resource_mut::<Audio<Source>>().unwrap();
+    let mut sinks = world.get_resource_mut::<Assets<AudioSink>>().unwrap();
 
     if let Some(audio_sources) = world.get_resource::<Assets<Source>>() {
-        audio_output.try_play_queued(&*audio_sources, &mut *audio);
+        audio_output.try_play_queued(&*audio_sources, &mut *audio, &mut *sinks);
     };
 }
+
+/// Asset controlling the playback of a sound
+///
+/// ```
+/// # use bevy_ecs::system::{Local, Res};
+/// # use bevy_asset::{Assets, Handle};
+/// # use bevy_audio::AudioSink;
+/// fn pause(
```

Comment:
It's not clear to me what this is trying to show. Is the intention of a system like that to be activated with a state change? I think adding some form of input in the example would make it clearer.

Reply:
The goal of the example is to show how to get the sink assets from an handle and that you can use it to pause or play the sound...
There are no access to input in `bevy_audio`, and setting up the whole state mechanism is quite out of scope of a code example in a doc comment

Reply:
Right, I just meant maybe adding a comment explaining that this would probably be toggled by a key press or some form of external event, because right now it just looks like a system that will constantly toggle between pausing and playing the music every frame. It's not super important I can still understand what's going on, I was just confused for a second when first reading it.

Reply:
added a comment 👍